coalib/settings/ConfigurationGathering.py

```diff
@@ -147,7 +147,13 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     for item in list(cli_sections["default"].contents.pop("targets", "")):
         targets.append(item.lower())
 
-    if bool(cli_sections["default"].get("no_config", "False")):
+    no_config_file_options = (
+        'no_config',
+        'show_bears',
+        'show_capabilities')
+
+    if any(option in cli_sections["default"]
```

Comment:
actually you should check `bool(cli_sections['default'].get(option, False))`, right? `no_config = False` could be the case.